addons/nuxeo-binary-metadata/src/main/java/org/nuxeo/binary/metadata/internals/MetadataMappingDescriptor.java

```diff
@@ -89,4 +93,8 @@ public String getBlobXPath() {
     public Boolean getIgnorePrefix() {
         return ignorePrefix;
     }
+
+    public Boolean getReadOnly() {
```

Comment:
What do you think about keeping the `Boolean` type for the field (which will allow merge for future usage) but rename this method `isReadOnly` and make it return `boolean`. For instance:
```
public boolean isReadOnly() {
    return Boolean.TRUE.equals(readOnly);
}
```

Reply:
I actually wanted to do that?. But then I noticed there already was this `getIgnorePrefix`, so I decided to keep it as `get` too, well, stay in "harmony". (I thought about renaming `getIgnorePrefix` to `ìgnorePrefix`, but thought I should not modify something that is not related to the ticket)

So, I'm ok with "is", want to make sure you're ok with having different styles in the class :-)

Reply:
Hey hey, you're totally right about  consistency. Here, what we would have done is to introduce `isIgnorePrefix` + deprecate `getIgnorePrefix` + replace everywhere former method name by the new one. I though it was too much to ask you for this contribution.

If you want to do it, it would be very cool ✌️. Otherwise, put the new name and we'll clean it one day 😉.

Reply:
ok, good. No no, it's not too much, i'll do it, thanks :-)